docs/support/tutorials/clean-up-data.md

```diff
@@ -104,7 +104,7 @@ Total size: 798343125192 bytes = 743.515 GiB = 0.726 TiB
 ```
 
 With this information, you are able to estimate how much time might be required to back up the
-data elsewhere, if you want to keep everything on the purge list outside of Puhti's Scratch file
+data elsewhere, if you want to keep everything on the purge list outside of Puhti's `scratch` file
```

Comment:
Do we monospace the names of the file systems as well? I would let it stay as "Scratch" when it's being referred to in text, while file paths would certainly be monospaced, like: `/scratch/...`

Let me know if I've missed a convention for this!

Reply:
Feel free to revert it back if you feel like it, otherwise this PR looks good to me!

Reply:
The _Disk areas_ page uses scratch, **scratch** and `scratch`. Personally, I'd prefer _scratch_. :smiley:

Reply:
Since you both have more review experience, I'll trust your judgement on this! I'm ready to merge this as it is :slightly_smiling_face: